src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const bodyOfTable = document.querySelector('tbody');
+
+const headOfTable = document.querySelector('tHead > tr');
+
+const tableRows = bodyOfTable.rows;
+
+headOfTable.addEventListener('click', () => {
+  let arrayByName;
+
+  switch (true) {
+    case event.target.cellIndex === 0:
+      arrayByName = [...tableRows].sort((a, b) => {
+        return a.children[0].innerText.localeCompare(b.children[0].innerText);
+      });
+      break;
+
+    case event.target.cellIndex === 1:
+      arrayByName = [...tableRows].sort((a, b) => {
+        return a.children[1].innerText.localeCompare(b.children[1].innerText);
+      });
+      break;
+
+    case event.target.cellIndex === 2:
+      arrayByName = [...tableRows].sort((a, b) =>
+        a.children[2].innerText - b.children[2].innerText);
+      break;
+
+    case event.target.cellIndex === 3:
+      arrayByName = [...tableRows].sort((a, b) => {
+        const first = parseFloat((a.children[3].innerText).slice(1));
```

Comment:
`parseFloat('138,575') = 138`, it's not that you want, choose another way of conversion

Reply:
> `parseFloat('138,575') = 138`, it's not that you want, choose another way of conversion

Done